packages/edit-site/src/components/dataviews/view-grid.js

```diff
@@ -50,7 +62,11 @@ export function ViewGrid( { data, fields, view, actions, getItemId } ) {
 									) ) }
 								</VStack>
 							</FlexBlock>
-							<FlexBlock>
+							{ /* TODO: ItemActions needs to be handled better in general.
+							In smaller viewports the actions could take too much space.
+							A solution could be to render the actions inside the media,
+							or in `grid` render all actions in the drop down menu. */ }
```

Comment:
Should we make these todo lists as tasks in the issue instead of leaving them in the code

Reply:
I'll do that. In general I was planning to work on follow ups right after this.